src/templates/chat/item-card.html

```diff
@@ -26,7 +26,7 @@ <h2>{{item.name}}</h2>
             {{/if}}
 
             {{#if data.save}}
-            <button data-action="save" data-save="{{data.save}}" disabled>
+            <button data-action="save" data-save="{{data.save}}">
```

Comment:
How does this button's event work? Does the selected token save, or the targeted token? Or the token selected/targeted at time of attack?

Reply:
Selected token